app/helpers/cms_helper.rb

```diff
@@ -14,4 +15,8 @@ def cms_image_url(image)
   def cms_image(image, class: nil)
     image_tag(cms_image_url(image), alt: image.alt, class:)
   end
+
+  def cms_colour_theme(colour, side)
```

Comment:
Should we add class to the name of this method, just to make it clear its returning a class name?